### ISSUE TITLE
what does `return_output_tensors` parameter do in inferflow_service.ini

### ISSUE THREAD

**Issue by ShadowTeamCN** [inferflow/inferflow#51]
And I set this argument to true, but not found PPL or logits in return
Besides, what is this argument supposed to return? Should it return only the logits of the generated output, the logits of the prompt, or both of it?
![image](https://github.com/inferflow/inferflow/assets/11573254/fa60c80c-b9df-4201-9c19-b810683bd2ae)
